content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +70,106 @@ env:
   value: "istio_ca/sign/istio-pki-role"
 {{< /text >}}
 
-## Deploy a testing workload
+## Deploy testing workloads
 
-This section deploys a testing workload `httpbin`. When the sidecar of the
+This section deploys testing workloads `httpbin` and `sleep`. When the sidecar of a
 testing workload requests a certificate through SDS, Node Agent will send
 certificate signing requests to Vault.
 
-1.  Generate the deployment for an example `httpbin` backend:
+1.  Generate the deployment for an example `httpbin` backend and an example `sleep` backend:
 
     {{< text bash >}}
     $ istioctl kube-inject -f @samples/httpbin/httpbin.yaml@ > httpbin-injected.yaml
+    $ istioctl kube-inject -f @samples/sleep/sleep.yaml@ > sleep-injected.yaml
     {{< /text >}}
 
-1.  Deploy the example backend:
+1.  Create a service account `vault-citadel-sa`:
 
     {{< text bash >}}
-    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl create serviceaccount vault-citadel-sa
     {{< /text >}}
 
-1.  List Node Agent's pods:
+1.  Edit the service account `vault-citadel-sa` to use an example JWT token that has been configured
```

Comment:
Can we explain more on this part? Customers may not know the limitation of our vault set up.

Reply:
Done: added more explanations.